vita3k/renderer/src/gl/draw.cpp

```diff
@@ -74,6 +76,14 @@ void draw(GLState &renderer, GLContext &context, GxmContextState &state, const F
         const std::string hash_text_v = hex_string(state.vertex_program.get(mem)->renderer_data->hash);
 
         LOG_DEBUG("\nVertex  : {}\nFragment: {}", hash_text_v, hash_text_f);
+
+        std::stringstream vert_ub("");
```

Comment:
```suggestion
        std::stringstream vert_ub;
```
it's already default-constructed